File: Programming_Task_03_ShaliniSengar/Python_Files/Password_Attempt_Simulator.py
password = "admin123"
attempts = 0

while attempts < 3:
    user_password = input("Enter Password: ")

    if user_password == password:
        print("Access Granted")
        break
    else:
        attempts += 1
        print("Incorrect Password")

if attempts == 3:
    print("Account Locked")

File: Programming_Task_03_ShaliniSengar/C Files/Password_Attempt_Simulator.c
#include <stdio.h>
#include <string.h>

int main() {
    char password[] = "admin123";   // Predefined password
    char input[50];
    int attempts;

    for (attempts = 1; attempts <= 3; attempts++) {
        printf("Enter password: ");
        scanf("%s", input);

        if (strcmp(input, password) == 0) {
            printf("Access Granted\n");
            return 0;
        } else {
            printf("Incorrect Password!\n");
        }
    }

    printf("Account Locked\n");

    return 0;
}